src/views/Matches.vue

```diff
@@ -158,8 +146,8 @@ export default defineComponent({
     onMounted(async (): Promise<void> => {
       await rankingsStore.retrieveSeasons();
       rankingsStore.setSeason(rankingsStore.seasons[0]);
-      matchStore.setSeason(rankingsStore.seasons[0]);
-      getMatches(1);
+      await matchStore.setSeason(rankingsStore.seasons[0]);
```

Comment:
Looks like this call already loads matches, so the line below would load them twice.

Reply:
Ah true, I just added the missing await as I was getting a warning. Will remove the redundant call